plugins/Tools.cc

```diff
@@ -377,10 +452,9 @@ void HHAnalyzer::fillTriggerEfficiencies(const Lepton & lep1, const Lepton & lep
         eff_lep2_leg1 = m_hlt_efficiencies.at("IsoMu17leg")->get(p_hlt_lep2)[0];
         eff_lep2_leg2 = m_hlt_efficiencies.at("IsoMu8orIsoTkMu8leg")->get(p_hlt_lep2)[0];
         DZ_filter_eff = DZ_filter_eff_MuMu;
-        // L1 EMTF bug
-        if (getMuonsSector(lep1.p4) == getMuonsSector(lep2.p4) && sameEndcap(lep1.p4, lep2.p4)) {
+        // FIXME L1 EMTF bug
+        if (isCSCSameSectorGen(lep1, lep2))
             DZ_filter_eff *= L1_EMTF_bug_eff_MuMu;
```

Comment:
shouldn't we apply reco for both data and mc ? We are creating an asymmetry of treatment here no ?